match_with_noaa_stations.py

```diff
@@ -149,12 +194,16 @@ def main():
             logging.info(
                 'closest NOAA to %s, is %s, with a distance of %s meters',
                 output['circle_name'],
-                output['closest_station_name'],
+                output['closest_station_id'],
                 output['distance'])
             results.append(output)
     df = pd.DataFrame.from_dict(results)
-    df.to_csv('closest_station.csv.gz', index=False)
+    df.to_csv('closest_stations.csv.gz', index=False)
+    return(df)
+
+# Run the main function to match
+#main(circles_data_smol,noaa_sations)
+closest_stations = main(cbc_usa,noaa_sations)
```

Comment:
We should reconsider this because a `main()` should not be passing in any parameters

Reply:
It takes a parameter now indicating if the NOAA stations should be collected from the API or a saved file. In the course of using this script there have been 3 or 4 iterations of using a saved file to match with. This script will use those file during any replication processes or any rematching activities.